docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -75,6 +75,9 @@ predictions are combined. The ensemble learning technique that we use in the
 {stack} is a type of boosting called extreme gradient boost (XGboost) which 
 combines decision trees with gradient boosting methodologies.
 
+{regression-cap} works as a batch analysis, if new data comes into your index, 
+you have to perform the {reganalysis} again on the altered data.
```

Comment:
> ...on the altered data.

This seems to imply that you can/should re-run the analysis only on the data that's changed. It doesn't specify how you'd make the job aware of which subset to re-analyze. Can we clarify?

If, on the other hand, you have to run re-run the job against the full set of data, this sentence should change to something like this: "If new data comes into your index, you must restart the {dfanalytics-job}".